gui/layers/_vectors_layer/view/controls.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# title           :
+# description     :
+# author          :bryant.chhun
+# date            :2/27/19
+# version         :0.0
+# usage           :
+# notes           :
+# python_version  :3.6
+
+from PyQt5.QtWidgets import QButtonGroup, QVBoxLayout, QRadioButton, QFrame
+
+from os.path import join
+from ....resources import resources_dir
+
+
+class QtVectorsControls(QFrame):
+    def __init__(self, layer):
+        super().__init__()
+
+        self.layer = layer
+        self.layer.events.mode.connect(self.set_mode)
+
+        # self.select_button = QtSelectButton(layer)
+        # self.addition_button = QtAdditionButton(layer)
+        self.panzoom_button = QtPanZoomButton(layer)
+
+        self.button_group = QButtonGroup(self)
+        # self.button_group.addButton(self.select_button)
+        # self.button_group.addButton(self.addition_button)
+        self.button_group.addButton(self.panzoom_button)
+
+        layout = QVBoxLayout()
+        # layout.addWidget(self.select_button)
+        # layout.addWidget(self.addition_button)
+        layout.addWidget(self.panzoom_button)
+        layout.addStretch(0)
+        self.setLayout(layout)
+        self.setMouseTracking(True)
+
+    def mouseMoveEvent(self, event):
+        self.layer.status = self.layer.mode
+
+    def set_mode(self, event):
+        mode = event.mode
+        # if mode == 'add':
+        #     self.addition_button.setChecked(True)
+        # elif mode == 'select':
+        #     self.select_button.setChecked(True)
+        if mode == 'pan/zoom':
+            self.panzoom_button.setChecked(True)
+        else:
+            raise ValueError("Mode not recongnized")
+
+
+class QtPanZoomButton(QRadioButton):
+    def __init__(self, layer):
+        super().__init__()
+
+        self.layer = layer
+        self.setToolTip('Pan/zoom mode')
+        self.setChecked(True)
+        styleSheet = button_style('zoom')
+        self.setStyleSheet(styleSheet)
+        self.toggled.connect(lambda state=self: self._set_mode(state))
+        self.setFixedWidth(28)
+
+    def _set_mode(self, bool):
+        with self.layer.events.mode.blocker(self._set_mode):
+            if bool:
+                self.layer.mode = 'pan/zoom'
+
+
+class QtSelectButton(QRadioButton):
```

Comment:
Can probably delete this as not in use right now. Do you have plans / desires for interactivity (line drawing etc.) or would that come in the shapes layer?

Reply:
don't have current plans but thought it'd be nice.  This can easily be added at a later date